.devops/pagopa-code-review-pipelines.yml

```diff
@@ -7,232 +7,228 @@
 #
 
 variables:
-  NODE_VERSION: '14.16.0'
+  NODE_VERSION: "14.16.0"
```

Comment:
it seems that there is no change here except for file spaces and apex, if so please restore this file